plugins/micromega/coq_micromega.ml

```diff
@@ -50,10 +50,10 @@ let get_lia_option () =
 let get_lra_option () =
  !lra_proof_depth
 
+(*let reify_tc = ref false*)
```

Comment:
What’s the purpose of this commented code? I should probably be removed.

Reply:
removed.

Reply:
removed